crates/language/src/language_settings.rs

```diff
@@ -75,6 +75,8 @@ pub struct LanguageSettings {
     pub hard_tabs: bool,
     /// How to soft-wrap long lines of text.
     pub soft_wrap: settings::SoftWrap,
+    // If syntax highlighting should work.
```

Comment:
```suggestion
    // Whether editor should perform syntax highlighting.
```

Let's also keep this phrase in other corners, e.g. crates/editor/src/element.rs and in default.json